FMS/wwwroot/css/site.css

```diff
@@ -77,10 +77,11 @@ body {
 -------------------------------------------------- */
 a:focus {
     color: #0056b3;
+    text-decoration: underline;
 }
 
 a:focus:not(.btn) {
-    text-decoration: underline;
+    text-decoration: none;
```

Comment:
What are you trying to fix with this change? This makes it hard to tell which menu item is selected when using the keyboard to navigate.

Reply:
Many hyperlinks became underlined that were not supposed to be (The "File Management System" header for one major one), so I changed this to work better. I can use this for specific instances instead of changing the site.css.